src/mocks/handlers.ts

```diff
@@ -0,0 +1,212 @@
+import { rest } from 'msw'
+
+export const handlers = [
```

Comment:
This file is all the mock responses for third party APIs so its possible to write tests without depending on them being up / changing values